cogs/clash_river_war.py

```diff
@@ -139,6 +139,30 @@ def remaining_war_embed(all_clan_attacks):
     embed.add_field(name=columns, value=row_val, inline=False)
     return embed
 
+def players_war_attacks(all_current_war_players: PlayersRemainingWarAttacks):
+    embed = discord.Embed(
+        description=dedent('''
+            **Current River Race Players War Attacks **

```

Comment:
Wouldn't a more appropriate name be "Current River Race - Unfinished war attacks players"? Or "Current River Race - Players with Unfinished War Attacks"?

Reply:
done